aiida/tools/dbexporters/tcod.py

```diff
@@ -875,8 +875,8 @@ def add_metadata_inline(what, node=None, parameters=None, args=None):
     for tag in node.values[dataname].keys():
         datablock[tag] = node.values[dataname][tag]
     datablocks.append(datablock)
-    for loop in node.values[dataname].loops:
-        loops[loop.keys()[0]] = loop.keys()
+    for loop in node.values[dataname].loops.values():
+        loops[loop[0]] = loop
 
```

Comment:
has the syntax really changed from keys to values?
I.e. is the result of this loop unchanged?

Reply:
yes (to both). Before, node.values[dataname].loops was a list, now its a dictionary, whose values correspond to the keys of the previous version (more or less...).

Reply:
Feels a bit loopy...
anyhow, thanks, just wanted to make sure.